build.sh

```diff
@@ -187,6 +187,16 @@ if buildAll || hasArg libcudf; then
 
     cmake --build . -j${PARALLEL_LEVEL} ${VERBOSE_FLAG}
 
+    # Record build times
+    if [[ -f "${LIB_BUILD_DIR}/.ninja_log" ]]; then
+        echo "Formatting build times"
+        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log > ${LIB_BUILD_DIR}/ninja_log.csv
+        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --xml > ${LIB_BUILD_DIR}/ninja_log.xml
+        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --html > ${LIB_BUILD_DIR}/ninja_log.html
```

Comment:
Build time xml report looks good (sortable too)
Why add html too?

Reply:
I don't really want to have these appear as test results. 
The xml was just a way for me to see the data and was intended to be removed once there was a way to see the html.
I'd like to make it more discoverable and easier to share/download.